### ISSUE TITLE
Call to a member function getStatusCode() on null

### ISSUE THREAD

**Issue by chicouskiluiz** [huzaifaarain/telescope-guzzle-watcher#11]
I've been using this package for a few weeks, but a few hours ago I started getting this error inside your package. To give you a context, the error is occurring within a job, which triggers my platform's webhooks. From what I noticed, this error has not yet occurred outside of webhooks. Do you have any idea what it could be?
Currently using these package versions:
```
"guzzlehttp/guzzle": "7.7.0",
"muhammadhuzaifa/telescope-guzzle-watcher": "2.0.1",
```
![image](https://github.com/user-attachments/assets/ed68db76-a415-4ad2-9fca-dc81ad8ccd71)

**Comment by huzaifaarain** [comment]
Hi @chicouskiluiz 

Thank you for using the package and providing me the feedback.

This can happen for various reasons such as host not reachable or any network error. This was patched in #10 Please upgrade your version.

Regards

**Comment by chicouskiluiz** [comment]
@huzaifaarain Thank you very much for the quick response! I actually had an older version, when I updated the problem was resolved.
